### DISCUSSION TITLE
Full screen mode on Mac app

### DISCUSSION THREAD

**Discussion by denisgl7** [airsdk/Adobe-Runtime-Support#818] [Q&A]
I have been using the following ane for a while to work with a full screen Mac app:
https://github.com/sushihangover/MacOSFullScreenANE
I think that any application should support the full screen feature.
This extension no longer works with the latest SDK. It is outdated. Will the default full screen feature be added to SDK?

**Comment by ajwfrost** [comment]
So presumably it's not working by setting the Stage.displayState property to one of the full screen modes? We will check this...
thanks

**Comment by denisgl7** [comment]
It turns out that can be without ANE, so everything is in order!
But there is also such a warning with a new SDK.
22: 45: 18.748 adl [15607: 884460] CoreText note: Client requested name "..SFCompactRounded-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage () or + [NSFont systemFontOfSize:].
What does it mean?

**Reply by ajwfrost** [reply]
Great, thanks for checking that.

The warning there isn't something I've seen in any of our tests but someone else mentioned it a while ago. But we don't have "SFCompactRounded" in our source code anywhere, I'm wondering whether this is something introduced by a tool (do you use Animate?) or just some default thing... are you able to create a small test case and send us the SWF/xml files so that we can try to reproduce this here?

thanks
